dts/arm/nxp/nxp_imx94x.dtsi

```diff
@@ -342,7 +568,181 @@
 				clocks = <&scmi_clk IMX943_CLK_ENET>;
 				status = "disabled";
 			};
+
+			emdio: mdio@4cde0000 {
+				compatible = "nxp,imx-netc-emdio";
+				reg = <0x4cde0000 0x10000>,
+					<0x4cbc0000 0x40000>;
+				reg-names = "basic", "pfconfig";
+				clocks = <&scmi_clk IMX943_CLK_ENET>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				status = "disabled";
+			};
+
+			netc_switch: switch@4cc00000 {
+				compatible = "nxp,netc-switch";
+				reg = <0x4cc00000 0x40000>,
+				      <0x4ca02000 0x1000>;
+				reg-names = "base", "pfconfig";
+				status = "disabled";
+				#address-cells = <1>;
+				#size-cells = <0>;
+
+				switch_port0: switch_port@0 {
+					compatible = "zephyr,dsa-port";
+					reg = <0>;
+					status = "disabled";
+				};
+
+				switch_port1: switch_port@1 {
+					compatible = "zephyr,dsa-port";
+					reg = <1>;
+					status = "disabled";
+				};
+
+				/* Parallel interface is muxed with enetc_psi0. */
+				switch_port2: switch_port@2 {
+					compatible = "zephyr,dsa-port";
+					reg = <2>;
+					status = "disabled";
+				};
+
+				/* Internal port */
+				switch_port3: switch_port@3 {
+					compatible = "zephyr,dsa-port";
+					reg = <3>;
+					ethernet = <&enetc_psi3>;
+					phy-connection-type = "internal";
+					dsa-tag-protocol = <DSA_TAG_PROTO_NETC>;
+					status = "disabled";
+				};
+			};
+		};
+
+		mu_m33s_secure_m71_for_m71: mbox@4d120000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d120000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+
+		mu_m33s_secure_m71_for_m33s: mbox@4d130000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d130000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+
+		mu_m33s_m71_for_m71: mbox@4d140000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d140000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+
+		mu_m33s_m71_for_m33s: mbox@4d150000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d150000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+
+		mu_m33s_secure_m70_for_m70: mbox@4d160000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d160000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+
+		mu_m33s_secure_m70_for_m33s: mbox@4d170000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d170000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
 		};
+
+		mu_m33s_m70_for_m70: mbox@4d180000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d180000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+
+		mu_m33s_m70_for_m33s: mbox@4d190000 {
+			compatible = "nxp,mbox-imx-mu";
+			reg = <0x4d190000 DT_SIZE_K(64)>;
+			#mbox-cells = <1>;
+			status = "disabled";
+		};
+	};
+	/* Define memory regions for IPC between m33s and m71 */
```

Comment:
For this and all the following IPC memory regions which are in DDR memory, per my understand they should could be customized, right? if yes, I suggest move them to board dts because they are not fixed value in SoC, they should be board level definitions according to external memory resource available.

Reply:
> For this and all the following IPC memory regions which are in DDR memory, per my understand they should could be customized, right? if yes, I suggest move them to board dts because they are not fixed value in SoC, they should be board level definitions according to external memory resource available.

There memory addresses are fixed value. If the board want to use different address. we can over write it in board level.